frontend/src/components/smart/CharacterDisplay.vue

```diff
@@ -130,7 +133,9 @@ export default {
       'ownCharacters',
       'timeUntilCharacterHasMaxStamina',
       'getIsInCombat',
-      'getIsCharacterViewExpanded'
+      'getIsCharacterViewExpanded',
+      'fightGasOffset',
+      'fightBaseline'
```

Comment:
fightGasOffset and fightBaseLine... I couldn't figure out how to use them in `methods: {` so had to pass from template html (as getters can be accessed only there) down to the methods.